src/pages/sign-up-index-page/page-proxy.js

```diff
@@ -0,0 +1,14 @@
+export const PageProxy = {
+  redirectToAdults() {
+    location.href += '/adults';
```

Comment:
here might be some problems if we want to reuse this class for form on adults landing page, which has url /courses/adults, so I'd suggest setting full path here, instead of adding.